src/SFA.DAS.AssessorService.Web/Startup.cs

```diff
@@ -143,40 +162,43 @@ private IServiceProvider ConfigureIoc(IServiceCollection services)
+                config.For<IStandardsApiClient>().Use<StandardsApiClient>().Ctor<ITokenService>().Is(c => c.GetInstance<ITokenService>("AssessorTokenService"));
+                config.For<IOppFinderApiClient>().Use<OppFinderApiClient>().Ctor<ITokenService>().Is(c => c.GetInstance<ITokenService>("AssessorTokenService"));
+                config.For<IDashboardApiClient>().Use<DashboardApiClient>().Ctor<ITokenService>().Is(c => c.GetInstance<ITokenService>("AssessorTokenService"));
+                config.For<IContactsApiClient>().Use<ContactsApiClient>().Ctor<ITokenService>().Is(c => c.GetInstance<ITokenService>("AssessorTokenService"));
+                config.For<ISearchApiClient>().Use<SearchApiClient>().Ctor<ITokenService>().Is(c => c.GetInstance<ITokenService>("AssessorTokenService"));
+                config.For<IEmailApiClient>().Use<EmailApiClient>().Ctor<ITokenService>().Is(c => c.GetInstance<ITokenService>("AssessorTokenService"));
+                config.For<IValidationApiClient>().Use<ValidationApiClient>().Ctor<ITokenService>().Is(c => c.GetInstance<ITokenService>("AssessorTokenService"));
+                config.For<ICertificateApiClient>().Use<CertificateApiClient>().Ctor<ITokenService>().Is(c => c.GetInstance<ITokenService>("AssessorTokenService"));
+                config.For<ILoginApiClient>().Use<LoginApiClient>().Ctor<ITokenService>().Is(c => c.GetInstance<ITokenService>("AssessorTokenService"));
+                config.For<IApplicationApiClient>().Use<ApplicationApiClient>().Ctor<ITokenService>().Is(c => c.GetInstance<ITokenService>("AssessorTokenService"));
+                config.For<ILearnerDetailsApiClient>().Use<LearnerDetailApiClient>().Ctor<ITokenService>().Is(c => c.GetInstance<ITokenService>("AssessorTokenService"));
+                config.For<IApprovalsLearnerApiClient>().Use<ApprovalsLearnerApiClient>().Ctor<ITokenService>().Is(c => c.GetInstance<ITokenService>("AssessorTokenService"));
+                config.For<ILocationsApiClient>().Use<LocationsApiClient>().Ctor<ITokenService>().Is(c => c.GetInstance<ITokenService>("AssessorTokenService"));
+                config.For<IStandardVersionClient>().Use<StandardVersionClient>().Ctor<ITokenService>().Is(c => c.GetInstance<ITokenService>("AssessorTokenService"));
+
+                config.For<IQnaApiClient>().Use<QnaApiClient>().Ctor<ITokenService>().Is(c => c.GetInstance<ITokenService>("QnATokenService"));
+                config.For<IRoatpApiClient>().Use<RoatpApiClient>().Ctor<ITokenService>().Is(c => c.GetInstance<ITokenService>("RoatpTokenService"));
+                config.For<IAzureApiClient>().Use<AzureApiClient>().Ctor<string>().Is(Configuration.AzureApiAuthentication.ApiBaseAddress);
 
+                config.For<IApiValidationService>().Use<ApiValidationService>().Ctor<ITokenService>().Is(c => c.GetInstance<ITokenService>("AssessorTokenService"));
```

Comment:
Since there are a dozen or so references to "AssessorTokenService", they can be replaced with a reference to a `private const string` instead of being hardcoded

Reply:
Done that and the same in the API